api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java

```diff
@@ -914,6 +928,29 @@ private void enableADT() {
                 findExtensionById(CustomAppLoaderConstants.EncounterTemplates.NO_DETAILS), "fas fa-fw fa-sign-out-alt", null, true, null, null);
     }
 
+    private void enableADTO3() {
```

Comment:
certainly open to a better name than "ADT O3", but nothing came to mind.

Reply:
maybe not using numbers in the name of the method?

Reply:
Is there an "ADT non-O3"?  If not, why not just ADT?

Reply:
NM.  I see below.  I'm not super concerned about the naming.